lib/amplitude.dart

```diff
@@ -29,13 +28,13 @@ class Amplitude {
   /// // If care about init complete
   /// await amplitude.isBuilt;
   /// ```
-  Amplitude(this.configuration, [MethodChannel? methodChannel]) {
+  Amplitude(Configuration configuration, [MethodChannel? methodChannel]) {
```

Comment:
As discussed, if you could revert the changes to hiding `Configuration` for now and leave in making `isBuilt` final I'd be happy to approve this and get this merged!